src/main/java/org/terasology/computer/display/component/DisplayComponent.java

```diff
@@ -62,4 +51,13 @@ public void setMode(String mode) {
     public void setData(List<String> data) {
         this.data = data;
     }
+
+    @Override
+    public void copyFrom(DisplayComponent other) {
+        this.monitorSize = other.monitorSize;
```

Comment:
As we always initialize the size we can reuse our own instance and just set the value.
```suggestion
        this.monitorSize.set(other.monitorSize);
```